src/TypeMapping/MysqliTypeMapper.php

```diff
@@ -49,7 +49,7 @@ public function __construct()
         }
     }
 
-    public function mapToPHPStanType(int $mysqlType, int $mysqlFlags, int $length): Type
+    public function mapToPHPStanType(int|string $mysqlType, int|array $mysqlFlags, int $length): Type
```

Comment:
When mysqlFlas is passed in as an the logic within the method will not work?

Reply:
yeah, sorry about this - I just wanted to see what things will be in common for the different mappers..

the logic is similar, but I'm not happy about the union types.. I'm thinking I'll drop the interface for now, or just drop the method definition and have the reflectors dependent of the specific mapper implementation...

Reply:
I am fine with dropping the interface, but my intial concerns were more about the duplicated logic of the 2 type-mappers.

but maybe you just need to play a bit longer and figure out, how this can be de-duplicated